hourly_watch.py

```diff
@@ -15,6 +15,50 @@
 
 
 # Ska-specific watchers
+class SkaURLWatch(JobWatch):
+    def __init__(self, task, maxage_hours, url=None,):
+        self.type = 'URL'
+        self.basename = url
+        super(SkaURLWatch, self).__init__(task, url, maxage=maxage_hours * HOURS)
+
+    @property
+    def headers(self):
+        if not hasattr(self, '_headers'):
+            try:
+                import urllib2
```

Comment:
Do you have suggestions for something more idiomatic to handle setting the values to something reasonable if the try does or does not work out?